src/Checkin.php

```diff
@@ -140,17 +140,21 @@ public function asRequestData(): array
     {
         $result = [
             'name' => $this->name,
-            'slug' => $this->slug,
             'schedule_type' => $this->scheduleType,
-            'report_period' => $this->reportPeriod,
-            'grace_period' => $this->gracePeriod,
-            'cron_schedule' => $this->cronSchedule,
-            'cron_timezone' => $this->cronTimezone,
+            'slug' => $this->slug ?? '',
```

Comment:
This change will cover the case where we want to send an update to unset the slug (or any of the optional values).